websockets/remote.go

```diff
@@ -598,3 +598,22 @@ func dump(b []byte) string {
 	out, _ := json.MarshalIndent(v, "", "  ")
 	return string(out)
 }
+
+// Load Gateway Balances for the given account
+// IMPORTANT: Check if the endpoint supports the command since not all endpoints support this command!
+// (last check xrplcluster.com supported the command)
+func (r *Remote) GatewayBalances(a data.Account) (*GatewayBalancesResult, error) {
+	cmd := &GatewayBalances{
+		Command:     newCommand("gateway_balances"),
+		Account:     a,
+		Strict:      true,
+		LedgerIndex: "validated",
+		HotWallet:   []data.Account{},
```

Comment:
Why send an empty JSON array for an optional parameter? Should this be a parameter of GatewayBalances?